vivisect/impemu/platarch/amd64.py

```diff
@@ -19,9 +19,9 @@ class Amd64WorkspaceEmulator(v_i_emulator.WorkspaceEmulator, e_amd64.Amd64Emulat
         e_amd64.REG_RDI, e_amd64.REG_R8,  e_amd64.REG_R9,
     ]
 
-    def __init__(self, vw, logwrite=False, logread=False):
+    def __init__(self, vw, logwrite=False, logread=False, stacksize=4096):
```

Comment:
yeah, i like this idea. after i got to the 5th or 6th constructor to add the same keyword parameter, i was thinking the same thing.

it cleans up the code in a few obvious ways:

  - the magic constant 4096 isn't sprinkled as many places.
  - the base emulator class has the optional keyword parameter stacksize, but because every subclass also has to have it, the default value for the base class is ignored. that makes me think that using keywords parameters is probably not the optimal solution.

i'll take a shot at implementing this as i have time this week.